reporter/gcppubsub/pubsub_test.go

```diff
@@ -0,0 +1,81 @@
+package gcppubsub
```

Comment:
Could we please also add a failing test for this? We want to make sure we don't throw a fatal or any disturbance on failing.